pkg/asset/agent/image/baseiso.go

```diff
@@ -117,7 +111,10 @@ func (i *BaseIso) Generate(dependencies asset.Parents) error {
 
 	var baseIsoFileName string
 	var err error
+	//agentManifests.ClusterImageSet = nil
 	if agentManifests.ClusterImageSet != nil {
+		rhcosArch := agentManifests.InfraEnv.Spec.CpuArchitecture
```

Comment:
The `rhcosArch := ` has to be change to `rhcosArch =` for global var assignment.